.github/workflows/tests.yml

```diff
@@ -141,6 +141,7 @@ jobs:
 
       - name: Setup indices
         run: |
+          php artisan es:index-scores:set-schema --schema=1
```

Comment:
the actual schema version should be defined using env instead so it always matches docker compose